shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/presentation/ui/FeedStatusItem.kt

```diff
@@ -48,11 +44,18 @@ fun FeedStatus(
     }
 }
 
-enum class FeedStatus(
-    @StringRes val titleId: Int,
-    @DrawableRes val iconId: Int,
-    val color: Color
-) {
-    GREEN(R.string.feed_status_green, R.drawable.ic_face_happy, CustomColor.StatusGreen),
-    RED(R.string.feed_status_red, R.drawable.ic_face_upset, CustomColor.StatusRed);
+@Composable
+@Preview
+fun FeedStatusItemGreenPreview() {
+    FeedStatusItem(
+        status = FeedStatus.GREEN
+    )
+}
+
+@Composable
+@Preview
+fun FeedStatusItemRedPreview() {
```

Comment:
It's better to put different previews of one small ui component in one preview using column and dividers